src/main/java/io/appform/dropwizard/actors/base/Handler.java

```diff
@@ -58,12 +70,19 @@ public void handleDelivery(String consumerTag, Envelope envelope,
                                AMQP.BasicProperties properties, byte[] body) throws IOException {
         try {
             final Message message = mapper.readValue(body, clazz);
-            boolean success = retryStrategy.execute(() -> handle(message, messageProperties(envelope)));
+            val tracer = GlobalTracer.get();
+            Span childSpan = buildChildSpan(properties, tracer);
+            log.info("publishing message with traceId: {}, spanId: {}", childSpan.context().toTraceId(),childSpan.context().toSpanId());
```

Comment:
this will lead to unnecessary logging

Reply:
changed logging level to debug

Reply:
removed logging since its an openSource bundle and we don't really log anything in other bundles also while starting a span